tools/c7n_azure/tests/test_tags.py

```diff
@@ -329,7 +337,9 @@ def test_must_specify_tags_to_remove(self):
     @arm_template('vm.json')
     @patch('c7n_azure.actions.utcnow', return_value=TEST_DATE)
     def test_auto_tag_add_creator_tag(self, utcnow_mock):
-        """Adds CreatorEmail to a resource group
+        """Adds CreatorEmail to a resource group.
+        If this test needs to be re-recorded, update TEST_DATE to
```

Comment:
I would move this comment up, because this matters for all auto_tag tests and tests that have to muck with the dates.